pkg/wallet/wallet.go

```diff
@@ -409,6 +404,33 @@ func (c *Wallet) resolveCredentials(verify bool, credentials ...CredentialToPres
 	return response, nil
 }
 
+func (c *Wallet) verifyCredential(credential json.RawMessage) (bool, error) {
+	// TODO resolve stored DIDs which can not be resolved by context?
```

Comment:
what's does this mean?

Reply:
We need custom VDRI for each wallet profile which also goes through stored DID in wallet content(cached DID Document). Part of my next PR. (ex: interop scenarios in wallet).

Reply:
fixed